### PR TITLE
Twitter Video Downloader

### LINKED ISSUE #1039: Twitter video downloader (GUI)
# Aim
Download videos from Twitter

# Details
The user will pass the link of the video in the text field and you need to implement the logic to download the video. If possible, implement the progress bar to show the progress of the download.

If progress bar, the level will be increased to 3 else 2.

### PR DESCRIPTION
**Pull request by XZANATOL**
# Description

Added Twitter video downloader project files. I figured a workaround for adding a detailed progress bar for the download. It's a bit of a personal preference to see detailed info of the download progress.

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] I have added the project meta data in the PR template.
- [x] I have created the ``requirements.txt`` file if needed.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [ ] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [x] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: Twitter Video Downloader

Folder: Twitter_Video_Downloader

Requirements: requirements.txt

Script: script.py

Arguments: none

Contributor: XZANATOL

Description: GUI App that downloads twitter video from its provided URL

### CODE REVIEW

File: Twitter_Video_Downloader/Readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Twitter Video Downloader
```
Comment (kaustubhgupta) [inline Twitter_Video_Downloader/Readme.md:1]: can we decrease all the heading levels after this one to level 2 (##). This is look good I guess

File: Twitter_Video_Downloader/Readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+A GUI used to download
```
Comment (kaustubhgupta) [inline Twitter_Video_Downloader/Readme.md:3]: A GUI used to download ... ? next?

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Initially there is a black screen with no text. this doesn't give a good look. Can you add some text for that, like: welcome to twitter downloader and then it disappears when the download process starts?

**Comment by XZANATOL** [comment]
You need to start the download for the progress to show up. xd

But I got your point, gonna make tge changes.

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍

**Review by santushtisharma10 — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]
Sorry for the delay. LGTM 👍🏻

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**